### PR TITLE
Fix "Getting Started with Qiskit" dead links

### LINKED ISSUE #647: "Getting Started" dead link to "Start Here"
### Steps to reproduce the problem
[Reported in Qiskit Slack](https://qiskit.slack.com/archives/CJG2E9LQP/p1591816135323100)
https://qiskit.org/documentation/getting_started.html#next-steps bullet list has a bad link to "Qiskit tutorials"
### What is the current behavior?
404
### What is the expected behavior?
https://github.com/Qiskit/qiskit-tutorials/blob/master/start_here.ipynb is the correct link.

### PR DESCRIPTION
**Pull request by filipecorrea**
### Summary

Fixes https://github.com/Qiskit/qiskit.org/issues/647

Fix documentation and tutorials links on "Getting Started with Qiskit" page:
- Qiskit documentation -> https://qiskit.org/documentation
- Qiskit tutorials -> https://github.com/Qiskit/qiskit-tutorials/blob/master/start_here.ipynb

### Details and comments

N/A.

### CODE REVIEW

**Comment by filipecorrea** [comment]
Travis build failure should not be related to this issue: `pyscf 1.7.4 has requirement scipy<1.5, but you have scipy 1.5.2.`

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Changelog](http://keepachangelog.com/en/1.0.0/).
 [UNRELEASED](https://github.com/Qiskit/qiskit/compare/0.12.1...HEAD)
 ====================================================================
 
+- Fixed documentation and tutorials links on "Getting Started with Qiskit" page.
+
```
Comment (SooluThomas) [inline CHANGELOG.md]: This need not be added.
```suggestion
```

File: docs/getting_started.rst [hunk 2]
```diff
@@ -252,7 +252,7 @@ Next Steps
 Now that you have learnt the basics, consider these learning resources:
 
 - `About the Qiskit framework <the_elements.html>`__
-- `Qiskit documentation <apidoc/qiskit.html>`__
-- `Qiskit tutorials <https://nbviewer.jupyter.org/github/Qiskit/qiskit-tutorials/blob/master/qiskit/1_start_here.ipynb>`__
+- `Qiskit documentation <https://qiskit.org/documentation>`__
+- `Qiskit tutorials <https://github.com/Qiskit/qiskit-tutorials/blob/master/start_here.ipynb>`__
```
Comment (SooluThomas) [inline docs/getting_started.rst]:
```suggestion
- `Qiskit tutorials <https://qiskit.org/documentation/tutorials/circuits/index.html>`__
```

File: docs/getting_started.rst [hunk 3]
```diff
@@ -252,7 +252,7 @@ Next Steps
 Now that you have learnt the basics, consider these learning resources:
 
 - `About the Qiskit framework <the_elements.html>`__
-- `Qiskit documentation <apidoc/qiskit.html>`__
-- `Qiskit tutorials <https://nbviewer.jupyter.org/github/Qiskit/qiskit-tutorials/blob/master/qiskit/1_start_here.ipynb>`__
+- `Qiskit documentation <https://qiskit.org/documentation>`__
```
Comment (SooluThomas) [inline docs/getting_started.rst]: I don't think this link has to be added here? @mtreinish

**Review by SooluThomas — CHANGES_REQUESTED** [review]

File: docs/getting_started.rst (hunk 3 above)
Comment (filipecorrea) [reply docs/getting_started.rst]: I agree with @SooluThomas. This link here seems redundant, but as it was broken, I decided to fix it. @mtreinish, please let me know if you agree with removing it.

Comment (mtreinish) [reply docs/getting_started.rst]: This does seem like a weird link to add, it's basically saying go back to the home page of the documentation which presumably is where you started to find this page. So I agree we should remove it. It'd be a different story if we had a front page for `apidoc` which is what I assume this was actually going for,

**Review by mtreinish — DISMISSED** [review]
Besides the inline comment we should not be using external html links for links to documentation. Instead we should use internal references: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role this way the links get generated correctly regardless of where the documentation is built or hosted.

**Comment by filipecorrea** [comment]
@mtreinish I got the internal link from [docs/index.rst](https://github.com/Qiskit/qiskit/blob/master/docs/index.rst). Can you please verify if that's the one you mentioned on https://github.com/Qiskit/qiskit/pull/992#pullrequestreview-461973819?

**Review by SooluThomas — APPROVED** [review]
LGTM! Thank you!

**Review by mtreinish — APPROVED** [review]

**Pull request merged**